File: generated-sources/dart/mojang-sessions/lib/model/player_texture.dart
part of openapi.api;

class PlayerTexture {
  
  PlayerSkinURL SKIN = null;
  
  PlayerTextureURL CAPE = null;
  PlayerTexture();

  @override
  String toString() {
    return 'PlayerTexture[SKIN=$SKIN, CAPE=$CAPE, ]';
  }

  PlayerTexture.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    SKIN = new PlayerSkinURL.fromJson(json['SKIN']);
    CAPE = new PlayerTextureURL.fromJson(json['CAPE']);
  }

  Map<String, dynamic> toJson() {
    return {
      'SKIN': SKIN,
      'CAPE': CAPE
    };
  }

  static List<PlayerTexture> listFromJson(List<dynamic> json) {
    return json == null ? new List<PlayerTexture>() : json.map((value) => new PlayerTexture.fromJson(value)).toList();
  }

  static Map<String, PlayerTexture> mapFromJson(Map<String, dynamic> json) {
    var map = new Map<String, PlayerTexture>();
    if (json != null && json.length > 0) {
      json.forEach((String key, dynamic value) => map[key] = new PlayerTexture.fromJson(value));
    }
    return map;
  }
}

File: generated-sources/csharp-dotnet2/mojang-sessions/src/main/CsharpDotNet2/Org/OpenAPITools/Model/PlayerTexture.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace Org.OpenAPITools.Model {

  /// <summary>
  /// Provide links to the player&#39;s skin and cape
  /// </summary>
  [DataContract]
  public class PlayerTexture {
    /// <summary>
    /// Gets or Sets SKIN
    /// </summary>
    [DataMember(Name="SKIN", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "SKIN")]
    public PlayerSkinURL SKIN { get; set; }

    /// <summary>
    /// Gets or Sets CAPE
    /// </summary>
    [DataMember(Name="CAPE", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "CAPE")]
    public PlayerTextureURL CAPE { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class PlayerTexture {\n");
      sb.Append("  SKIN: ").Append(SKIN).Append("\n");
      sb.Append("  CAPE: ").Append(CAPE).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}
